e2e/generate-fixtures-json.js

```diff
@@ -18,11 +18,14 @@ const async = require('async');
 // ]
 
 let username;
+let sha = 'HEAD';
```

Comment:
This won't work when running `yarn e2e` local. I resist to call out the git binary to get the latest sha, because this doesn't work under all conditions for example the latest commit isn't pushed yet.

Reply:
Maybe we should consider not adding tests for relative links in workflow files. What do you think @xt0rted ?

Reply:
I'm ok with that. Should I remove that test and rebase this or do you want to merge it with the fixup commits?

Reply:
I think using an env var doesn't solve the actual issue, because the sha will be different all the time, right? Also this would fail when running on a fork right? Let's just remove that test and get this merged.

Reply:
When these run they pull whatever is in the current branch, so the sha will change on each run. If the test only ran when `GITHUB_ACTION` was `true` then we'd also be able to get the sha from `GITHUB_SHA`, but that won't work with the current test setup. Maybe in the future, but not now.

Reply:
A while ago I came across the fact that the env var `GITHUB_SHA` is different depending on whether your workflow runs on push or pull_request events.

Pull request event:  Last merge commit on the GITHUB_REF branch
Push event: Commit pushed, unless deleting a branch (when it's the default branch)

https://help.github.com/en/actions/reference/events-that-trigger-workflows